smarty/environment.py

```diff
@@ -226,90 +262,50 @@ def asSMIRKS(self):
             """
             Returns
             --------
-            the same as asSMARTS()
-                for consistency asSMARTS() or asSMIRKS() can be called
-                for all environment objects
+            smarts : str
+                The SMIRKS string for just this bond
             """
+            #the same as asSMARTS()
+            #    for consistency asSMARTS() or asSMIRKS() can be called
+            #    for all environment objects
             return self.asSMARTS()
 
-        def addORtype(self, ORtype):
-            """
-            Adds ORtype to the set for this bond.
-
-            Parameters
-            --------
-            ORtype: string
-                added to the list of ORtypes for this bond
-            """
-            self.ORtypes.append(ORtype)
-
-        def addANDtype(self, ANDtype):
-            """
-            Adds ANDtype to the set for this bond.
-
-            Parameters
-            --------
-            ANDtype: string
-                added to the list of ANDtype for this bond
-            """
-            self.ANDtypes.append(ANDtype)
-
-        def getORtypes(self):
-            """
-            returns a copy of the list of ORtypes for this bond
+        def getOrder(self):
```

Comment:
We haven't put it into use, but Christopher and I wanted the ability to check valence around an atom as a possible way of proposing a move is valid. I'm not convinced it is necessary because we won't keep SMIRKS strings that don't type any molecule, but the methods exist if we want to be able to check them. 

The idea is that really you want the minimum bond order associated with a bond if you're going to filter by valence. for example if a bond is [*:1]-,=[*:2] the single or double bond should only count as 1 for bond order. The same idea goes for != or !: the minimum bond order for those is 1. the getOrder can then be used to get the minimum bond order around an atom in an environment.